src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,22 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
```

Comment:
One again, in reset css - we remove unneeded styles.
In normalize - add some specific styles